.github/dependabot.yml

```diff
@@ -0,0 +1,64 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'weekly'
+      # Use Eastern Standard Time (UTC -05:00)
+      timezone: 'America/Toronto'
+    open-pull-requests-limit: 1
+    commit-message:
+      prefix: 'chore: '
+    reviewers:
+      - 'Seneca-CDOT/winter-2021-telescope'
+      - 'c3ho'
+      - 'manekenpix'
```

Comment:
You can put me in here.  But before we put these folks in, let's make sure they want to be flagged for reviews.  cc @cindyledev, @manekenpix, @raygervais, @c3ho.

Reply:
Alternatively, you could create another team under `Seneca-CDOT` organization to include everyone who we want to be notified about dependency updates for Telescope @humphd

Reply:
I made a "Telescope Maintainers" team you can use instead of these names directly.

Reply:
Perfect!

Reply:
@humphd, just to clarify, should we keep `winter-2021-telescope` team as well?

Reply:
I'd do the maintainers only

Reply:
Alright